docs/Running-behind-nginx.md

```diff
@@ -114,8 +152,8 @@ Of course you can add as many backend servers as you like.
 
 Using the above, you access Netdata on the backend servers, like this:
 
-- `http://nginx.server/netdata/server1/` to reach `backend-server1`
-- `http://nginx.server/netdata/server2/` to reach `backend-server2`
+- `http://netdata.example.com/netdata/server1/` to reach `backend-server1`
+- `http://netdata.example.com/netdata/server2/` to reach `backend-server2`
 
 ### Using TLS communication
```

Comment:
Try to think of something nicer for this, that is also using the verb consistently with the other titles below. e.g. 'Limit direct access to Netdata' and 'Encrypt the communication between Nginx and Netdata', or both turned to 'Limiting' and 'Using'.
Perhaps you'll leave it as just 'Use TLS communication', though I think we put the wrong title here. We have a link to this header further down in the document as well, the build process will warn you if you leave something out.